conf/outputs.conf

```diff
@@ -1,32 +1,26 @@
-# Configuration for influxdb server to send metrics to
-# this is an example,  of what can be done.   Not included in docker build.
-[[outputs.influxdb]]
+[[outputs.influxdb_v2]]
+  ## The URLs of the InfluxDB cluster nodes.
+  ##
+  ## Multiple URLs can be specified for a single cluster, only ONE of the
+  ## urls will be written to each interval.
+  ##   ex: urls = ["https://us-west-2-1.aws.cloud2.influxdata.com"]
   urls = ["${INFLUXURL}"]
-  database = "telegraf"
 
-  database_tag = "db_tag"
-  exclude_database_tag = true
+  ## API token for authentication.
+  token = "${INFLUXDB_TOKEN}"
 
-  ## Retention policy to write to. Empty string writes to the default rp.
-  retention_policy = ""
-  ## Write consistency (clusters only), can be: "any", "one", "quorum", "all"
-  write_consistency = "any"
+  ## Organization is the name of the organization you wish to write to; must exist.
+  organization = "${INFLUXDB_ORG}"
 
-  ## Write timeout (for the InfluxDB client), formatted as a string.
-  ## If not provided, will default to 5s. 0s means no timeout (not recommended).
-  timeout = "5s"
-
-# [[outputs.file]]
-#   ## Files to write to, "stdout" is a specially handled file.
-#   files = ["stdout"]
+  ## Destination bucket to write into.
+  bucket = "${INFLUXDB_BUCKET}"
 
-#   ## Data format to output.
-#   ## Each data format has its own unique set of configuration options, read
-#   ## more about them here:
-#   ## https://github.com/influxdata/telegraf/blob/master/docs/DATA_FORMATS_OUTPUT.md
-#   data_format = "json"
+  ## The value of this tag will be used to determine the bucket.  If this
+  ## tag is not set the 'bucket' option is used as the default.
+  bucket_tag = "db_tag"
 
-# [[outputs.prometheus_client]]
-#   ## Address to listen on.
-#   listen = ":9273"
```

Comment:
I think the commented out plugins were here to serve as an example, and we should keep it (unless they don't work anymore).